### PR TITLE
Add test for defined directory

### PR DESCRIPTION
**Pull request by PiotrKowalski**

### CODE REVIEW

File: bin/test_piotr.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import requests
+f = open('../test/queries/english_00.txt', 'r')
```
Comment (sudheesh001) [inline bin/test_piotr.py]: Can you use this to read any of the files present in the `queries` directory, read each file and fire queries for each query in the file. That way its a generic script to fire anything in the `queries` directory.

Comment (yasoob) [inline bin/test_piotr.py]: @sudheesh001 I have submitted a PR with exactly the kind of functionality you are talking about :+1:

File: bin/test_piotr.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (sudheesh001) [inline bin/test_piotr.py]: The python `shebang` is missing here, Could you please add that?

**Comment by sudheesh001** [comment]
@PiotrKowalski I've figured out the issue you were facing. The problem is that you're not converting the response to a `json()` , Requests by default encapsulates the json responses. So here is the work around
```python
for line in current_file:
		url = "http://localhost:9000/api/search.json?q=" +  line
		response = requests.get(url)
		x = response.json()
		try:
			print(x['statuses'])
		except Exception as e:
			print(e)
```

**Comment by sudheesh001** [comment]
@PiotrKowalski Please squash everything into one single commit !

**Comment by sudheesh001** [comment]
Please resolve conflicts and squash :smile:

**Comment by PiotrKowalski** [comment]
I am trying to do it :P

**Comment by Orbiter** [comment]
@sudheesh001 can you please test this and merge it, I am currently out of office. Thank you!

**Comment by sudheesh001** [comment]
@Orbiter Tested :smile:

**Pull request merged**